bgcalendar.js

```diff
@@ -77,7 +77,10 @@ var bgcalendar = {};
         d1.setFullYear(year, month - 1, date);
         d2.setFullYear(year, 0, 1);
         
-        return Math.floor((d1.getTime() - d2.getTime()) / DAY);
+        var millis = d1.getTime() - d2.getTime();
+        var rem = millis % DAY;
+        var dayNumber = ((millis - rem ) / DAY )  +   (rem > 0 ? 1 : 0);
+        return dayNumber;
```

Comment:
Instead of `Math.floor` you can use `Math.ceil`, so `Math.ceil(86.5)` will return 87.